mui/dockwidgets/state_list_widget.py

```diff
@@ -46,6 +49,30 @@ def __init__(self, name: str, parent: ViewFrame, data: BinaryView):
         self.states: Dict[int, StateDescriptor] = {}
         self.state_items: Dict[int, QTreeWidgetItem] = {}
 
+    @Slot(QTreeWidgetItem, int)
+    def onClick(self, item: QTreeWidgetItem, col: int):
+        """Jump to the current PC of a given state when double clicked"""
+
+        # do nothing on non-state items
+        if item in self.state_lists:
+            return
```

Comment:
Is there a better way to do this? I'm thinking something like
```python
if item not in self.state_items:
    return
```
This way we only operate on positively identified state items instead of checking that the item is only not in `self.state_lists`, which would have to be modified if we add some other items

Reply:
By storing state id in `QTreeWidgetItem`s, it also helps differentiate state items and non-state items.

```python
item_id = item.data(0, StateListWidget.STATE_ID_ROLE)

# do nothing on non-state items
if item_id is None:
    return
```